pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -72,11 +72,17 @@ const (
 	signozTraceTableName       = "distributed_signoz_index_v2"
 	signozTraceLocalTableName  = "signoz_index_v2"
 	signozMetricDBName         = "signoz_metrics"
-	signozSampleLocalTableName = "samples_v2"
-	signozSampleTableName      = "distributed_samples_v2"
-	signozTSTableName          = "distributed_time_series_v2"
-	signozTSTableNameV4        = "distributed_time_series_v4"
-	signozTSTableNameV41Day    = "distributed_time_series_v4_1day"
+	signozSampleLocalTableName = "samples_v4"
+	signozSampleTableName      = "distributed_samples_v4"
+
+	signozTSLocalTableNameV4 = "time_series_v4"
```

Comment:
can you use SIGNOZ_TIMESERIES_v4_6HRS_LOCAL_TABLENAME from constants ?

Reply:
I will do another PR.